tests/e2e/online_tests.py

```diff
@@ -264,109 +266,95 @@ def test_delete_projects(self):
 
     @pytest.mark.order(10)
     def test_wb_publish(self):
-        name_on_server = OnlineCommandTest.TWBX_WITH_EXTRACT_NAME
         file = os.path.join("tests", "assets", OnlineCommandTest.TWBX_FILE_WITH_EXTRACT)
-        arguments = self._publish_args(file, name_on_server)
+        arguments = self._publish_args(file, OnlineCommandTest.TWBX_WITH_EXTRACT_NAME)
         _test_command(arguments)
 
-    @pytest.mark.order(10)
+    @pytest.mark.order(11)
     def test_wb_get(self):
+        # add .twbx to the end to tell the server what we are getting
         self._get_workbook(OnlineCommandTest.TWBX_WITH_EXTRACT_NAME + ".twbx")
 
-    @pytest.mark.order(10)
+    @pytest.mark.order(11)
     def test_view_get_pdf(self):
         wb_name_on_server = OnlineCommandTest.TWBX_WITH_EXTRACT_NAME
         sheet_name = OnlineCommandTest.TWBX_WITH_EXTRACT_SHEET
         self._get_view(wb_name_on_server, sheet_name + ".pdf")
 
-    @pytest.mark.order(10)
+    @pytest.mark.order(11)
     def test_view_get_csv(self):
         wb_name_on_server = OnlineCommandTest.TWBX_WITH_EXTRACT_NAME
         sheet_name = OnlineCommandTest.TWBX_WITH_EXTRACT_SHEET
         self._get_view(wb_name_on_server, sheet_name + ".csv")
 
-    @pytest.mark.order(10)
+    @pytest.mark.order(11)
     def test_view_get_png(self):
         wb_name_on_server = OnlineCommandTest.TWBX_WITH_EXTRACT_NAME
         sheet_name = OnlineCommandTest.TWBX_WITH_EXTRACT_SHEET
         self._get_view(wb_name_on_server, sheet_name + ".png")
 
-    @pytest.mark.order(11)
-    def test_wb_delete(self):
-        name_on_server = OnlineCommandTest.TWBX_WITH_EXTRACT_NAME
-        self._delete_wb(name_on_server)
-
     @pytest.mark.order(11)
     def test_wb_publish_embedded(self):
-        name_on_server = OnlineCommandTest.TWB_WITH_EMBEDDED_CONNECTION
+        pytest.skip("waremart failed to establish a connection to your datasource.")
         file = os.path.join("tests", "assets", OnlineCommandTest.TWB_WITH_EMBEDDED_CONNECTION)
```

Comment:
Does this line need to change to `EMBEDDED_TWB_NAME` too?

Reply:
No, this line wants the filename, not the document name.